share/spice/population/population.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_population = function(api_result){
+
+        if (!api_result || !api_result[1][0].value) {
+            return Spice.failed('population');
+        }
+
+        var result = api_result[1][0],
+            population  = result.value,
+            countryName = result.country.value,
+            lastUpdated = result.date;
+
+        var script = $('[src*="/js/spice/population/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/population\/([^\/]+)\/([^\/]+)/);
+
+        
+        var query_country = decodeURIComponent(query[2]);
+        var URL_country = countryName.replace(/\,|\./g,"").replace(/ /g,"-").replace(/Dem/g,"democratic").replace(/Rep/g,"republic").toLowerCase();
+        
+        population = population.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
Awesome, much better solution than what I had.
Are the rest of these DDG functions documented anywhere? I could not find them anywhere.